docs/UserGuide.adoc

```diff
@@ -310,6 +312,7 @@ Examples:
 * `theme`
 * `t`
 
+// tag::undoredo[]
```

Comment:
Can anyone help to confirm if having multiple instances of the same tag in the guide will result in any linking issues (perhaps from the personal portfolio for starters?)